indico/htdocs/js/indico/jquery/colorswitch.js

```diff
@@ -0,0 +1,114 @@
+                    {bgColor: '#FFEC1F', textColor: '#1F1D04'}
+                ], [
+                    {bgColor: '#DFEBFF', textColor: '#0F264F'},
+                    {bgColor: '#0D316F', textColor: '#EFF5FF'},
+                    {bgColor: '#1A3F14', textColor: '#F1FFEF'},
+                    {bgColor: '#5F171A', textColor: '#FFFFFF'},
+                    {bgColor: '#D9DFC3', textColor: '#272F09'}
+                ], [
+                    {bgColor: '#4F144E', textColor: '#FFEFFF'},
+                    {bgColor: '#6F390D', textColor: '#FFEDDF'},
+                    {bgColor: '#8ec473', textColor: '#021F03'},
+                    {bgColor: '#92b6db', textColor: '#03070F'},
+                    {bgColor: '#DFDFDF', textColor: '#151515'}
+                ], [
+                    {bgColor: '#ecc495', textColor: '#1F1100'},
+                    {bgColor: '#b9cbca', textColor: '#0F0202'},
+                    {bgColor: '#C2ECEF', textColor: '#0D1E1F'},
+                    {bgColor: '#d0c296', textColor: '#000000'},
+                    {bgColor: '#EFEBC2', textColor: '#202020'}
+                ]
+            ]
+        },
+
+        _create: function() {
+            var self = this;
+            var element = self.element;
+            var opt = self.options;
+            var palette = $('<div></div>', {'class': 'color-palette', 'style': 'display: none'});
+            var paletteTable = $('<table></table>');
+            var trigger = self.element.find('.switch-trigger');
+
+            for (var i = 0; i < opt.defaultColors.length; ++i) {
+                var tr = $('<tr></tr>', {'style': 'height: 13px'});
+                for (var j = 0; j < opt.defaultColors[i].length; ++j) {
+                    var td = $('<td></td>', {
+                        'style': 'cursor: pointer; background: ' + opt.defaultColors[i][j].bgColor,
+                        'class': 'palette-color'
+                    });
+
+                    var colorBox = $('<div></div>', {
+                        'style': 'text-align: center;',
+                        'value': opt.defaultColors[i][j].bgColor,
+                        'class': 'background-box'
+                    });
+
+                    colorBox.append($('<div></div>', {
+                        'style': 'background: ' + opt.defaultColors[i][j].textColor,
+                        'class': 'text-box',
+                        'value': opt.defaultColors[i][j].textColor
+                    }).css('margin', 'auto').width('8px').height('8px'));
+
+                    td.on('click', function() {
+                        var $this = $(this),
+                            backgroundColor = $this.find('.background-box').val(),
+                            textColor = $this.find('.text-box').val();
+
+                        trigger.css('background', backgroundColor)
+                               .attr('style', trigger.attr('style') + 'color: ' + textColor + ' !important');
```

Comment:
I don't remember why but I had some problems with the same expressions. However I will give it a try again.